jerry-core/jmem/jmem-heap.c

```diff
@@ -148,8 +150,8 @@ void
 jmem_heap_init (void)
 {
 #ifndef JERRY_CPOINTER_32_BIT
-  JERRY_STATIC_ASSERT (((UINT16_MAX + 1) << JMEM_ALIGNMENT_LOG) >= JMEM_HEAP_SIZE,
-                       maximum_heap_size_for_16_bit_compressed_pointers_is_512K);
+  /* maximum_heap_size_for_16_bit_compressed_pointers_is_512K */
+  JERRY_ASSERT (((UINT16_MAX + 1) << JMEM_ALIGNMENT_LOG) >= JMEM_HEAP_SIZE);
```

Comment:
because JMEM_HEAP_SIZE maybe not constant, so I use JERRY_ASSERT instead of JERRY_STATIC_ASSERT